Gettextf fixes

FransMeerhoff opened the pull request:
Cleanup gettextf - Phase 1 : Remove unicode characters as parameters

Commit:
Update Dutch translation files

After some Audit changes

Commit:
Cleanup gettextf - Phase 1 : Remove unicode characters as parameters

- Phase 1 : Remove unicode characters as parameters

TimKDJ commented on JASP-Engine/JASP/po/R-nl.po:
```diff
@@ -754,17 +754,19 @@ msgstr ""
+msgid ""
+"The point estimate of the true population value <i>W&#770 = N × w&#772</i>"
+msgstr ""
+"De puntschatting van de ware populatiewaarde <i>W&#770 = N × w&#772</i>"
 
 msgid "The uncertainty of the estimator %1$s%2$s%3$s"
 msgstr "De onzekerheid van de schatter %1$s%2$s%3$s"
 
-msgid "The confidence interval around the point estimate %1$s"
-msgstr "Het betrouwbaarheidsinterval rond de puntschatting %1$s"
+msgid "The confidence interval around the point estimate <i>W&#770 ± U</i>"
+msgstr "Het betrouwbaarheidsinterval rond de puntschatting <i>W&#770 ± U</i>"
```
Are these correct?

FransMeerhoff replied:
The confidence interval around the point estimate Ŵ ± U
Het betrouwbaarheidsinterval rond de puntschatting Ŵ ± U

The point estimate of the true population value Ŵ = N × w̄
De puntschatting van de ware populatiewaarde Ŵ = N × w̄
 
I do see the italic in the real result:

![Screen Shot 2020-05-20 at 09 45 57](https://user-images.githubusercontent.com/15656960/82419481-c7fc9f00-9a7e-11ea-8634-fb6ce4ee81e6.png)

So I think it is correct

TimKDJ replied:
Might be good to change the html characters to unicode too
`W&#770` is `\U0174` for example

Commit:
Cleanup gettextf: Phase 2: Remove decimal unicode literals from parameters

- Cleanup gettextf: Phase 2: Remove literals from parameters list
- Remove Unicode Decimal code like &#770 to unicode code points like /u304

TimKDJ commented on JASP-Engine/JASP/R/anovarepeatedmeasures.R:
```diff
@@ -1391,7 +1391,7 @@ AnovaRepeatedMeasures <- function(jaspResults, dataset = NULL, options) {
     
   } else {
-    thisOverTitle <- gettextf("95%% CI for Mean Difference")
+    thisOverTitle <- gettext("95% CI for Mean Difference")
```
Double `%` are still a necessity as well as the `gettextf()`

FransMeerhoff replied:
Yes, I just did that yesterday. I was believing in a miracle i think because in 
gettext("95% bca\u002A CI") anovarepeatedmeasure.R 1387 it works fine. I though that once you found that part of parsing that went wrong. Because this is very annoying too I think.

TimKDJ replied:
See my explanation here: https://github.com/jasp-stats/INTERNAL-jasp/issues/777#issuecomment-597740225
It will only work when the translation and original text share the same "percent parameters". So it's just luck of the draw if it works.

Commit:
Cleanup gettextf: Phase 3: remove literals from parameter list

- remove literals from parameter list, mostly html tags

Commit:
Update Dutch translation regarding file handling (Import/Export)

- After ace7a95fbd69b8da06c5b4fac2d7435945daba5c some Dutch translations disappeared.
